src/main/java/com/redhat/red/build/koji/model/converter/KojiBuildComponentConverter.java

```diff
@@ -53,7 +53,7 @@ public Object render( BuildComponent value )
             map.put( NAME, rpm.getName() );
             map.put( VERSION, rpm.getVersion() );
             map.put( RELEASE, rpm.getRelease() );
-            map.put( EPOCH, rpm.getEpoch() );
+            map.put( EPOCH, rpm.getEpoch() != null ? rpm.getEpoch().toString() : null );
```

Comment:
I left the JSON type as `String`. Or should we fix it here, but potentially cause more breakage?

Reply:
String looks safe.